ROADMAP.md

```diff
@@ -68,4 +68,14 @@ event-based waiter fallbacks to the old way of block monitoring with
 `block_added` notifications subscription.
 
 Removal of stale RPC server compatibility code from `waiter.EventWaiter` is
-scheduled for Jun-Jul 2024 (~0.107.0 release).
\ No newline at end of file
+scheduled for Jun-Jul 2024 (~0.107.0 release).
+
+## Removal of Dump*Slot() methods of `vm.Context`
```

Comment:
> Removal of Dump*Slot() methods of `vm.Context`

Just `Dump*Slot() methods of vm.Context`, because it's already in the "Deprecated functionality" section.